### PR TITLE
Fix full tests

### PR DESCRIPTION
**Pull request by h-mayorquin**
Full tests are failling. I think this is related to this issue here:

https://github.com/int-brain-lab/ibllib/issues/753

### CODE REVIEW

**Review by alejoe91 — APPROVED** [review]
Thanks for looking into this @h-mayorquin

File: pyproject.toml [hunk 1]
```diff
@@ -86,7 +88,7 @@ full = [
     "h5py",
     "pandas",
     "xarray",
-    "scipy",
+    "scipy<1.13",
```
Comment (samuelgarcia) [inline pyproject.toml:91]: maybe for full we could remove this boundarie no ?
is it only for ibl stuff ?

Comment (h-mayorquin) [reply pyproject.toml:91]: I guess we could move the boundary to the testing section. 

We gain:
* Some people will be able to use the latest scipy with other packages that don't use the wrong import.
We lost:
Some people will install full and then it won't work with IBL.

Your call guys, @samuelgarcia @alejoe91

Comment (samuelgarcia) [reply pyproject.toml:91]: upper boundaries is sometimes anoying.
For me, the balance is that very few people use the ibllib reader vs more poeple are making a fresh install with recent scipy.
lets discuss this in a call

**Pull request merged**